components/raftstore/src/store/local_metrics.rs

```diff
@@ -155,6 +155,9 @@ pub struct RaftMetrics {
     pub propose: RaftProposalCounterVec,
     pub invalid_proposal: RaftInvalidProposalCounterVec,
     pub raft_log_gc_skipped: RaftLogGcSkippedCounterVec,
+    pub read_indx_coal: LocalIntCounter,
```

Comment:
Consider not abbreviating these words. Abbreviating `index` to `indx` saves only one letter (our), and `coal` lacks comprehensibility.